### PR TITLE
Release 2023-10-08

### PR DESCRIPTION
**Pull request by alvinli222**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+  * 
+
+* Behavioral Changes
+  * With the [release](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#10042023--) of Container Insights 3.1.14, default 1-year tokens will be set to 1-hour expiry and refreshed at 10 minutes.
+  * A warning has been added for clusters utilizing dual-stack networking and outbound type user-defined routing if the associated route table does not have a default IPv6 route in place.
+  * All control plane upgrades to AKS 1.26+ will enforce deprecated API usage validation. You can bypass this check and frontend pdb validation with [forceUpgrade](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli#stop-cluster-upgrades-automatically-on-api-breaking-changes-preview)
+
+* Component Updates
+  * Windows CNI has been updated to v1.4.39.1 for Azure CNI Overlay and Azure CNI with dynamic allocation.
+  * Azure Monitor Metrics for AKS has been updated. Please see their [release notes](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md?plain=1#L3) for full details.
```
Comment (palma21) [inline CHANGELOG.md:25]: updated to which image version?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+  * Microsoft Defender has been updated to support for kernel version 6.2 or higher.
```
Comment (palma21) [inline CHANGELOG.md:26]: New image version?

Comment (palma21) [reply CHANGELOG.md:26]: Should this be a bugfix having in mind kernel 6.2 has been out for a bit?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+* Features
+  * 
+
+* Preview Features
+  * 
+
+* Behavioral Changes
+  * With the [release](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#10042023--) of Container Insights 3.1.14, default 1-year tokens will be set to 1-hour expiry and refreshed at 10 minutes.
+  * A warning has been added for clusters utilizing dual-stack networking and outbound type user-defined routing if the associated route table does not have a default IPv6 route in place.
```
Comment (palma21) [inline CHANGELOG.md:20]: How/Where is that warning displayed? At cluster create? 

Do we have an entry in the docs for this, @chasewilson

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+  * All control plane upgrades to AKS 1.26+ will enforce deprecated API usage validation. You can bypass this check and frontend pdb validation with [forceUpgrade](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli#stop-cluster-upgrades-automatically-on-api-breaking-changes-preview)
```
Comment (palma21) [inline CHANGELOG.md:21]: That link still contains preview. Update missing? LRB finished?
@shashankbarsin / @wenjungaogaogao

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (alvinli222) [reply CHANGELOG.md:24]: not quite sure how this works but there is no docs from MDC team. will followup for confirmation.

File: CHANGELOG.md (hunk 1 above)
Comment (alvinli222) [reply CHANGELOG.md:23]: will follow up.

File: CHANGELOG.md (hunk 4 above)
Comment (alvinli222) [reply CHANGELOG.md]: LRB is finished.

File: CHANGELOG.md (hunk 3 above)
Comment (chasewilson) [reply CHANGELOG.md:21]: Have a [PR here](https://github.com/MicrosoftDocs/azure-docs-pr/pull/254841) to put a note about default route. The error is just to let customers know as it's not a requirement and could be an issue for outbound traffic. Don't prevent customers from creating cluster as there are cases where IPv6 traffic is internal only and IPv4 is fine for outbound.

**Review by qpetraroia — APPROVED** [review]

**Review by chasewilson — APPROVED** [review]

**Review by wenjungaogaogao — APPROVED** [review]

**Review by charleswool — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (charleswool) [reply CHANGELOG.md]: Added link to the doc for detail reference

**Review by charleswool — APPROVED** [review]

**Comment by charleswool** [comment]
sign-off

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+
+* Features
+  * Support customizing probe config when *externalTrafficPolicy* = *local* and *health_probe_port* annotation is defined for service port. [Cloud Provider docs link](https://cloud-provider-azure.sigs.k8s.io/topics/loadbalancer/#custom-load-balancer-health-probe-for-port)
```
Comment (palma21) [inline CHANGELOG.md:13]: Azure docs link for supported probes?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+  * Load Balancer TCP reset can now be disabled with a service annotaiton *"service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset=true"* in preview. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4520)
```
Comment (palma21) [inline CHANGELOG.md:14]: Azure doc link for supported probes

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
...
+
+* Bug Fixes
+  * Microsoft Defender for Containers has been updated to image version 1.3.81 to support kernel versions 6.2 or higher.
+
+* Behavioral Changes
+  * With the [release](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md#10042023--) of Container Insights 3.1.14, default 1-year tokens will be set to 1-hour expiry and refreshed at 10 minutes.
+  * A warning has been added for clusters utilizing dual-stack networking and outbound type user-defined routing if the associated route table does not have a default IPv6 route in place. Visit [Dual-stack kubenet networking](https://learn.microsoft.com/en-us/azure/aks/configure-kubenet-dual-stack) for full details.
+  * All control plane upgrades to AKS 1.26+ will enforce deprecated API usage validation. You can bypass this check and frontend pdb validation with [forceUpgrade](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli#stop-cluster-upgrades-automatically-on-api-breaking-changes).
```
Comment (palma21) [inline CHANGELOG.md:24]: Feature GA, is LRB done?

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (charleswool) [reply CHANGELOG.md:13]: This change hasn't yet been updated in Azure doc, probably need to remove

File: CHANGELOG.md (hunk 6 above)
Comment (charleswool) [reply CHANGELOG.md:14]: This change hasn't yet been updated in Azure doc, probably need to remove

File: CHANGELOG.md (hunk 7 above)
Comment (charleswool) [reply CHANGELOG.md:24]: Yes, the LRB is cleared: https://dev.azure.com/msazure/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/462515/GA-API-Breaking-change-detection-and-bypassing-through-force-upgrade-on-minor-upgrades

**Review by charleswool — APPROVED** [review]

**Review by palma21 — APPROVED** [review]

**Pull request merged**